src/constants.ts

```diff
@@ -21,4 +21,13 @@ export const APP_STAKING_LOCKUP_DURATION_MS = 30 * 1000 // 30 seconds
 
 export const SUPPORTED_AINFT_STANDARDS = {
   721: '721',
-}
\ No newline at end of file
+}
+
+export const Path = {
```

Comment:
It doesn't seem constants tho, what about moving it into util.ts?

Reply:
Done!